src/gobexport/__main__.py

```diff
@@ -1,9 +1,5 @@
-from gobexport.app import run as run_app
+from gobexport import app
 
-
-def init():
-    if __name__ == "__main__":
-        run_app()
-
-
-init()
+if __name__ == "__main__":
+    # pragma: no cover
```

Comment:
# pragma: no cover should be after if __name__ == clause to exclude everything in if clause from coverage